src/conventional/amdar_bufr2ioda.py

```diff
@@ -0,0 +1,514 @@
+#!/usr/bin/python
+
+from datetime import datetime, timedelta
+import dateutil.parser
+import os
+from pathlib import Path
+import sys
+import time
+import logging
+
+import numpy as np
+import netCDF4 as nc
+from eccodes import CODES_MISSING_LONG, codes_bufr_new_from_file, codes_set, codes_get_array, KeyValueNotFoundError, codes_release
```

Comment:
is this line not too long for coding norms? Alternative to this approach is to do:
`import eccodes as ecc` and then each of these calls can be `ecc.CODES_MISSING_LONG` for example.